app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -47,7 +48,7 @@
 
     <% unless omit_header %>
       <%= render "govuk_publishing_components/components/layout_header", {
-        environment: "public",
+        environment: environment,
```

Comment:
Do we anticipate anyone needing this argument in a public setting? I'm expecting not so I'm wondering if we want to just remove this argument until a need arises.

Reply:
Good point, have removed, thanks.